app/models/concerns/parameter_search.rb

```diff
@@ -0,0 +1,36 @@
+module ParameterSearch
+  extend ActiveSupport::Concern
+
+  included do
+    scoped_search :relation => parameter_relation_symbol, :on => :searchable_value, :in_key => parameter_relation_symbol, :on_key => :name, :rename => :params, :complete_value => true, :only_explicit => true, :operators => ['= ', '~ '], :ext_method => :search_by_params
+  end
+
+  module ClassMethods
+    def search_by_params(key, operator, value)
+      key_name = key.sub(/^.*\./, '')
+      parameters_relation = parameter_relation_symbol
+
+      conditions = sanitize_sql_for_conditions(
+        ["parameters.name = ? and parameters.searchable_value #{operator} ?", key_name, value_to_sql(operator, value)]
+      )
+      build_query = unscoped
```

Comment:
what about authorization? or would that be added anyways by the controller after the search scope?

Reply:
Yes, at controller level it uses `resource_base` which calls `authorized` if that method exists on model.